nestor_api/lib/io.py

```diff
@@ -100,3 +111,9 @@ def remove(file_path):
             os.remove(file_path)
         else:
             raise
+
+
+def write(file_path: str, content: str) -> None:
+    """ Write the string content into the file at the given path"""
+    file = open(file_path, "w")
+    file.write(content)
```

Comment:
Same comment as above about closing and the `with` structure.

Also, you may need to use `"w+"` to create the file if it does not exists (not a 100% sure on this one).

Reply:
From what I read, I can see that `+` is only to read as well when opening with w (for writing).